tests/test_data_pipeline.py

```
@@ -0,0 +1,92 @@
+
+        assert "train" in raw
+        assert "validation" in raw
+        assert "test" in raw
+        assert len(raw["train"]) > 0
+
+    @pytest.mark.slow
+    def test_prepare_with_sample_limit(self, config):
+        config.data.max_train_samples = 20
+        config.data.max_val_samples = 5
+        pipeline = WikiSQLPipeline(config)
+        train, val, test = pipeline.prepare()
```

Ruff (0.15.1)

[warning] Unpacked variable `test` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)